dev_guide/managing_images.adoc

```diff
@@ -586,6 +586,235 @@ instead of listing all image streams.
+Administrators can create, view, and delete all tokens associated an
+organization.
+
+//[[Prerequisites]]
+*Prerequisites*
+
+* User credentials
+
+//[[Procedure]]
+*Procedure*
+
+To create a token in order complete a `docker login`:
+
+. Navigate to `registry.redhat.io`.
+. Log in with your RHN username and password.
+. Accept terms when prompted.
+.. If you are not immediately prompted to accept terms, you will be prompted
+when proceeding with the following steps.
+. From the *Registry Service Accounts* page, click *Create Service Account*
+.. Provide a name for the service account. It will be prepended with a random string.
+.. Enter a description.
+.. Click create.
+. Navigate back to your Service Accounts.
+. Click the Service Account you created.
+. Copy the username, including the prepended string.
+. Copy the token.
+
+[[authenticate-against-idp]]
+=== Authenticating Against IDP
+
+Once you create user credentials, you can use them to authenticate with the Red
+Hat SSO identity provider (IDP).
+
+//[[Prerequisites]]
+*Prerequisites*
+
+* User credentials
+
+//[[Procedure]]
+*Procedure*
+
+To run the authentication call:
+
+. Run a basic HTTP Authentication:
++
+* With user credentials:
++
+----
+# curl -u <$USERNAME:$PASSWORD>
+"https://sso.$ENV.redhat.com/auth/realms/rhcc/protocol/redhat-docker-v2/auth?service=docker-registry&client_id=curl&scope=repository:rhel:pull"
+----
++
+* With authentication token:
++
+----
+# curl -u <$TOKENID:$SECRET>
+"https://sso.$ENV.redhat.com/auth/realms/rhcc/protocol/redhat-docker-v2/auth?service=docker-registry&client_id=curl&scope=repository:rhel:pull"
+----
++
+. Run OAuth Direct Access Grant:
++
+* With user credentials:
++
+----
+# curl -X POST --data
+'username=$USERNAME&password=$PASSWORD&grant_type=password&service=docker-registry&client_id=curl-test&scope=repository:rhel:pull' https://sso.$ENV.redhat.com/auth/realms/rhcc/protocol/redhat-docker-v2/auth
+----
++
+* With authentication token:
++
+----
+# curl -X POST --data
+'username=$TOKENID&password=$SECRET&grant_type=password&service=docker-registry&client_id=curl-test&scope=repository:rhel:pull' https://sso.$ENV.redhat.com/auth/realms/rhcc/protocol/redhat-docker-v2/auth
+----
+
+[[docker-login]]
+=== Using Docker Login to Access the New Registry
+
+//[[Prerequisites]]
+*Prerequisites*
+
+* User credentials
+* Docker login access
+
+//[[Procedure]]
+*Procedure*
+
+To acquire the correct credentials in your {product-title} namespace to access
+the registry:
+
+. Log in to the new registry with a `docker login`.
++
+* Log in with user credentials:
++
+----
+# docker login -u <$USERNAME> registry.<$ENV>.redhat.io
+password: [$PASSWORD]
+----
++
+* Log in with authentication token:
++
+----
+# docker login -u <$TOKENID> registry.<$ENV>.redhat.io
+password: [$PASSWORD]
+----
++
+. Pull images:
++
+----
+# docker pull registry.dev.redhat.io/rhel
+----
+
+[[setting-registry-as-default]]
+=== Setting the New Registry as default
```

Comment:
i think you can delete this whole section too.  it's not really openshift specific nor is it necessary to make things work.